core/src/main/java/io/grpc/ChannelImpl.java

```diff
@@ -198,9 +205,11 @@ public ChannelImpl shutdown() {
    *
    * <p>NOT YET IMPLEMENTED. This method currently behaves identically to shutdown().
    */
-  // TODO(ejona86): cancel preexisting calls.
   public ChannelImpl shutdownNow() {
     shutdown();
+    for (ClientCall<?, ?> call : activeCalls) {
+      call.cancel();
```

Comment:
Still not seeing it.  The ClientCallImpl calls cancel from ~some~ thread.

Also, thats dumb that "cancels the call" cannot invoke call.cancel().